### ISSUE TITLE
Listing notes linking to the current note

### ISSUE THREAD

**Issue by wbgt** [notable/notable#957]
### Feature description

When a note is viewed or edited, a list of the notes linking to the current note can be shown (in side bar or by click a button), as we can see in Tiddlywiki or Scrivener.

### Feature motivation

This feature will promote Notable a personal knowledge system.

**Comment by fabiospampinato** [comment]
We could add a command for this in the future, but we would need to start collecting some metadata about this in order to implement this feature in a performant way.

**Comment by wbgt** [comment]
Thanks for your quick reply.
Since Notable can link to searching result (that is great), I guess a easy way to this feature is by designing a searching syntax for the internal links.

**Comment by fabiospampinato** [comment]
Yes that wouldn't be a problem. Actually on a second thought once we switch to UUIDs #538 this can be implemented in a much more performant way than I had thought.
